lib/src/feeds.dart

```diff
@@ -308,7 +308,7 @@ class Feed implements Comparable<Feed> {
 }
 
 class FeedManager {
-  final http.Client httpClient = new http.Client();
```

Comment:
For now, can you leave in the `new` keyword?

Reply:
Sure!

Reply:
79e0c9d